### PR TITLE
Documentation for security vulnerability reporting in FAQ

### PR DESCRIPTION
**Pull request by Richard-Hansen**
Jira ticket: https://ucsc-cgl.atlassian.net/browse/SEAB-2123

Added documentation for reporting security vulnerabilities.

Also will be updating Discourse pinned ticket, https://discuss.dockstore.org/t/opening-helpdesk-tickets/1506.

Currently will be updating the first sentence to:
```To send messages directly to administrators in the case of privacy complaints, security vulnerabilities, or any other urgent matter related to Dockstore, you can utilize the Discourse mailing feature.```

### CODE REVIEW

**Review by coverbeck — COMMENTED** [review]
I thought we were going with an email for security incidents?

**Comment by Richard-Hansen** [comment]
> I thought we were going with an email for security incidents?

I thought it would be good to have the current setup documented in any case. The Discourse method is decent as-is... but perhaps I jumped the gun.

For context, the 'competing' ticket is https://ucsc-cgl.atlassian.net/browse/SEAB-2279.

**Review by coverbeck — APPROVED** [review]
> I thought it would be good to have the current setup documented in any case. The Discourse method is decent as-is... but perhaps I jumped the gun.

Ah OK, makes sense.

**Review by NatalieEO — APPROVED** [review]

**Review by GFJHogue — APPROVED** [review]

**Review by garyluu — APPROVED** [review]

File: docs/faq.rst [hunk 1]
```diff
@@ -314,6 +314,24 @@ follow these steps:
...
+
+The following steps can be taken to create a helpdesk ticket (also shown `here <https://discuss.dockstore.org/t/opening-helpdesk-tickets/1506>`_).
+
+1. Navigate to `Discourse <https://discuss.dockstore.org/>`_ and login.
+2. Select your profile icon, located in the top right corner of the screen.
+3. Select the ``mail`` icon, located in the dropdown.
+4. Send a message to the ``admins`` group.
+
+.. Note:: If you are unable to see a ``New Message`` button on the mail page, you may be considered a new user and have insufficient privileges. \
+   Viewing 30 topics over 10 minutes will raise your privileges. You will be notified of any privilege changes to your account via the mailbox.
```
Comment (denis-yuen) [inline docs/faq.rst:332]: The default is just 5 topics over 10 minutes 
https://blog.discourse.org/2018/06/understanding-discourse-trust-levels/

**Review by denis-yuen — APPROVED** [review]

**Pull request merged**